docs/editors/overview.md

```diff
@@ -4,6 +4,21 @@ title: Text Editors
 sidebar_label: Overview
 ---
 
+## Latest Metals server versions
```

Comment:
I feel like it will be much easier to find this way. Alternatively we could move it to the top of each editor?

Reply:
I think this is fine as long as we don't remove it from the editor page since I feel like some people just skip the overview page and go right to their editor.